static/app/views/settings/components/dataScrubbing/modals/form/index.tsx

```diff
@@ -150,6 +147,7 @@ class Form extends Component<Props<Values, KeysOfUnion<Values>>, State> {
                 onChange={this.handleChange('pattern')}
                 value={values.pattern}
                 onBlur={onValidate('pattern')}
+                aria-label={t('Regular expression')}
```

Comment:
wanna fix the ID thing since it's open again haha

Reply:
Did you put it on both the control and the field?

Reply:
oh I see... I updated the code. thanks!